astropy/tests/coveragerc

```diff
@@ -7,11 +7,10 @@ omit =
    astropy/*/tests/*
    astropy/tests/test_*
    astropy/extern/*
-   astropy/sphinx/*
    astropy/utils/compat/*
    astropy/version*
-   astropy/wcs/docstrings*
```

Comment:
If I remember correctly, this one is generated and should be excluded from coverage? Not sure about `astropy/sphinx`.

Reply:
astropy/sphinx has been a directory back in 2012 when the line was last touched. After that arguably I haven't checked docstrings, so will double check it now.

Reply:
arrg, you're right, I'm adding back docstrings, sorry about the oversight